### DISCUSSION TITLE
Throttle trame-vtk view render events on window resize

### DISCUSSION THREAD

**Discussion by banesullivan** [Kitware/trame#585] [Q&A]
One really awesome feature of trame-vtk is the ability to resize the browser window and have the trame-vtk view and thus VTK render window dynamically resize with it. However on slower internet connections or in some scenarios, these resize events can get delayed causing significant lag that builds up and happens all at once (maybe my websocket connection is busy with other things and the resize messages are held back?). Either way, I'm wondering if it is possible to turn off the dynamic resizing of the render window or throttle the requests made to resize it?

Does anyone else notice this issue? I'll try to capture a screen recording if I notice it again

**Comment by jourdain** [comment]
I think throttling it make sense. Usually with trame we throttle on network/server busyness so it will automatically adapt to your environment. Right now the implementation is handled at the vtk.js side... I guess we will need to add more option and use another channel to sync the size. While not that difficult, it will require some time/funds to properly enable it.
